en/components/components-overview.md

```diff
@@ -15,7 +18,8 @@ The Sketch library has a `🧩 Components` page containing all Components and sy
 All **Indigo.Design System** Components are built on top of rigid [styling foundations](../style/styling-overview.md) letting you leverage an elaborate mechanism for theming and branding your user interface.
 
 > [!WARNING]
-> Unlinking any component instance from the library makes it impossible to automatically apply updates from subsequent versions of the Design System and may impact the code generation capability of the component. The code generation services rely on specially designated `🚫 metadata` layers, containing metadata descriptions of the state, template and other properties of the component.
+> Detaching a component from the Sketch library makes it impossible to automatically apply updates from subsequent versions of the Design System and may impact the code generation capability of the component. The code generation services rely on specially designated `🚫 metadata` layers, containing metadata descriptions of the state, template and other properties of the component.
+> Ungrouping a component or pattern from the Adobe XD library has even worse implications because it turns all nested components into groups and breaks the code generation service for the component as it cannot trace correctly its structure. You can make any state, layout and styling adjustment in Adobe XD without ungrouping a component so we kindly advise you to keep the entirety of the components on your artboards.
```

Comment:
@YoannaIvanova could you add this second part as a warning in the patterns overview as well?

Reply:
Added because of duplication in the PR https://github.com/IgniteUI/indigo-design-docfx/pull/528